Makefile

```diff
@@ -100,7 +100,6 @@ build: BUILD_ARGS=-o $(BUILDDIR)/
 
 $(BUILD_TARGETS): check_version go.sum $(BUILDDIR)/
 	GOWORK=off go $@ -mod=readonly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
-
```

Comment:
What about this part of the spec in the issue?

"source the path of the env to .profile: . "$HOME/.osmosisd/env""

Reply:
Refer from [cmt](https://github.com/osmosis-labs/osmosis/pull/3787#issuecomment-1362194375), I move this step at the same time as the init chain